webapp/packages/core-utils/src/getProgressPercent.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+export function getProgressPercent(done: number, total: number) {
```

Comment:
this function is duplicate of the same fn in the webapp/packages/plugin-navigation-tree-file-system/src/NavigationTreeFileContextMenuService.ts

(please reuse this new one)

Reply:
I will